### PR TITLE
add gmm, klarmobil, spritpreise & tgtg

### PR DESCRIPTION
**Pull request by peterge1998**
## Description

Fixes #(issue)

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:x: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories, such as copyediting)

### Icons added
* App Name (`package.name`) - added from icon request form
* App Name (`package.name`)
* App Name (`package.name`)

### Icons updated
* App Name (`package.name`)
* App Name (`package.name`)

### Icons linked
* App Name (linked `package.name` to `@drawable/package`)
* App Name (linked `package.name` to `@drawable/package`)

### CODE REVIEW

**Comment by peterge1998** [comment]
What do I miss? It reports too_good_to_go_foreground is missing, but icon request says this is the line I have to add:
` <item component="ComponentInfo{com.app.tgtg/com.app.tgtg.activities.login.SplashActivity}" drawable="too_good_to_go"/>`

**Comment by x9136** [comment]
@peterge1998

I think you should delete **too_good_to_go_foreground.svg** and add "o" to the svg file name, because you have a drawable in appfilter.xml that says the svg should be called **too_good_to_go**, but the svg file is called **to_good_to_go**.

**Comment by peterge1998** [comment]
> @peterge1998
> 
> I think you should delete **too_good_to_go_foreground.svg** and add "o" to tgtg svg file name, because you have a drawable in appfilter.xml that says the svg should be called **too_good_to_go**, but the svg file is called **to_good_to_go**. A typo.

fcking typo :) thx!

**Review by Chefski — CHANGES_REQUESTED** [review]
`too_good_to_go.svg` and `klarmobilde.svg` need to have their strokes changed to whole numbers

**Comment by peterge1998** [comment]
> `too_good_to_go.svg` and `klarmobilde.svg` need to have their strokes changed to whole numbers

Yep, you are correct. I always forget to set to whole numbers when I resize the paths at the end to 160x160.
I fixed klarmobil to 12 px.
But tgtg "needs" to stay at 7.5 for the right circle, otherwise the width does not look even compared to the left & middle circle:
image

**Comment by Chefski** [comment]
The icon already has a bump. Either the icon will be changed to fit the guidelines or it can't be merged

image

**Comment by peterge1998** [comment]
> The icon already has a bump. Either the icon will be changed to fit the guidelines or it can't be merged
> image

KK, done, its all 8 px now.

File: svgs/too_good_to_go.svg
Comment (Chefski) [inline svgs/too_good_to_go.svg:1]: Two paths still have 8.03px as the stroke

**Review by Chefski — APPROVED** [review]

**Pull request merged**